configs/agent.yml

```diff
@@ -116,3 +116,11 @@ runtime:
     - type: prometheus.time
     - type: prometheus.uname
     - type: prometheus.vmstat
+
+discovery:
+  hints:
+    # systemd: string, regular expression to use when discovering nodes run by systemd
+    systemd: metrikad-(flow|solana).service
+
+    # docker: string, regular expression to use when discovering nodes run by Docker
+    docker: consensus_1-1
```

Comment:
Any config changes should also reflect in the ansible folder.

Reply:
Addressed in ddaa36a.